.github/workflows/goBuild.yml

```diff
@@ -8,15 +17,37 @@ jobs:
       matrix:
         go: ['1.18', '1.19']
```

Comment:
Do we need the matrix here?

Reply:
We don't, but we'll make this parameterizable in a future PR.